docs/content/preview/deploy/multi-dc/async-replication/async-deployment.md

```diff
@@ -608,3 +608,35 @@ Alters involving adding/removing columns or modifying data types require replica
     ```output
     Replication enabled successfully
     ```
+#### Adding a column with a non-volatile default value
+
+When adding a new column with a (non-volatile) default expression, make sure to perform the schema modification on the target with the **computed** default value. 
+
+Example:
+
+Let's say we have a replicated table `test_table`.
+1. Pause replication
+1. Execute add column command on the source:
+   ```sql
+   ALTER TABLE test_table ADD COLUMN test_column TIMESTAMP DEFAULT NOW()
+   ```
+1. Perform the above `ALTER TABLE` command with the computed default value on the target:
+   
+    - The computed default value can be retrieved from `pg_attribute.attmissingval`.
```

Comment:
what is "pg_attribute.attmissingval". Maybe some detail about it can be helpful?

Reply:
how about something like this:

The computed default value can be retrieved from the `attmissingval` column in the `pg_attribute` catalog table.